judgels-backends/uriel/uriel-app/src/main/java/judgels/uriel/contest/scoreboard/ContestScoreboardResource.java

```diff
@@ -79,4 +83,12 @@ public ContestScoreboardResource(
                             .build();
                 });
     }
+
+    public void refreshScoreboard(Optional<AuthHeader> authHeader, String contestJid) {
+        String actorJid = actorChecker.check(authHeader);
+        Contest contest = checkFound(contestStore.getContestByJid(contestJid));
+        checkAllowed(scoreboardRoleChecker.canSupervise(actorJid, contest));
```

Comment:
```suggestion
        checkAllowed(scoreboardRoleChecker.canManage(actorJid, contest));
```

(please add that method + test)